lte/gateway/deploy/agw_install.sh

```diff
@@ -11,12 +11,30 @@ KVERS=$(uname -r)
 MAGMA_VERSION="${MAGMA_VERSION:-v1.3}"
 GIT_URL="${GIT_URL:-https://github.com/magma/magma.git}"
 
+
+
 echo "Checking if the script has been executed by root user"
 if [ "$WHOAMI" != "root" ]; then
   echo "You're executing the script as $WHOAMI instead of root.. exiting"
   exit 1
 fi
 
+wget https://raw.githubusercontent.com/magma/magma/"$MAGMA_VERSION"/lte/gateway/deploy/agw_pre_check.sh
+if [[ -f ./agw_pre_check.sh ]]; then
+  chmod 644 agw_pre_check.sh && bash agw_pre_check.sh
```

Comment:
it would be better to change the file permission of this file in git repo to avoid it setting during every install.

Reply:
Do you know how to do that?  I have no clue how @pshelar

Reply:
@tmdzk you can use `chmod` cmd in your local repo and commit the change. git tracks file permissions.

Reply:
Is this PR still warm?

If so, then I will follow up on this as part of my AGW work on Ubuntu20VM.